src/main/java/com/amazon/opendistroforelasticsearch/sql/plugin/SqlPlug.java

```diff
@@ -47,6 +48,9 @@
 
 public class SqlPlug extends Plugin implements ActionPlugin {
```

Comment:
seems there is inconsistent indentation in this file

Reply:
> seems there is inconsistent indentation in this file

Yes, it's a long pending issue...

Reply:
Will fix this later to avoid big PR with unrelated changes. AFAIK, we do have a few files with this problem.